cmd/gtas/cli/gtas.go

```diff
@@ -308,6 +308,20 @@ func (gtas *Gtas) fullInit() error {
 	common.GlobalConf.SetString(Section, "miner", gtas.account.Address)
 	fmt.Println("Your Miner Address:", gtas.account.Address)
 
+	//set the time for proposer package
+	timeForPackage := common.GlobalConf.GetInt(Section, "time_for_package", 2000)
+	if timeForPackage > 100 && timeForPackage < 2000 {
+		core.ProposerPackageTime = time.Duration(timeForPackage) * time.Microsecond
```

Comment:
should be time.Duration(timeForPackage) * time.Millisecond!

Reply:
ok